addons/iap/tools/iap_tools.py

```diff
@@ -147,11 +154,12 @@ def iap_jsonrpc(url, method='call', params=None, timeout=15):
         raise exceptions.ValidationError(
             _('The request to the service timed out. Please contact the author of the app. The URL it tried to contact was %s', url)
         )
+    except exceptions.ValidationError:
+        _logger.warning('And error occurred with the URL %s:', url)
```

Comment:
Why changing the log level from `exception` to `warning` ?
It feels like sweeping the exception under the rug.